releases/NS30_GP/6.11/Makefile

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2019 Richard Hughes <richard@hughsie.com>
+# Copyright (C) 2019 Mikhail f. Shiryaev <mr.felixoid@gmail.com>
+
+VENDOR=8Bitdo
+PROJECT_NAME=NS30_GP
```

Comment:
This should be SN30_GP, not NS30_GP. (The directory needs to be renamed too.)

Reply:
Thank you